src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -311,11 +311,16 @@ method java.util.Map clear
 method java.util.Map containsKey java.lang.Object
 method java.util.Map entrySet
 method java.util.Map get java.lang.Object
+method java.util.Map getOrDefault java.lang.Object java.lang.Object
 method java.util.Map isEmpty
 method java.util.Map keySet
 method java.util.Map put java.lang.Object java.lang.Object
 method java.util.Map putAll java.util.Map
+method java.util.Map putIfAbsent java.lang.Object java.lang.Object
 method java.util.Map remove java.lang.Object
+method java.util.Map replace java.lang.Object java.lang.Object
+method java.util.Map replace java.lang.Object java.lang.Object java.lang.Object
+method java.util.Map replaceAll java.lang.Object
```

Comment:
Looks like the test failure is because this method doesn't exist. There is `Map.replaceAll(BiFunction)`, but I'm not sure how well java's lambda functions interact with Groovy.

Reply:
I decided to remove it - java lambdas do not play well with Groovy.